src/include/target.h

```diff
@@ -35,11 +35,11 @@ typedef uint32_t target_addr_t;
 typedef struct target_controller target_controller_s;
 
 #if PC_HOSTED == 1
-uint32_t bmp_swd_scan(uint32_t targetid);
-uint32_t bmda_jtag_scan(void);
+bool bmp_swd_scan(uint32_t targetid);
```

Comment:
This  should probably be renamed to `bmda_swd_scan()` for consistency

Reply:
I did wonder why it was named like this